dll/win32/comctl32/commctrl.c

```diff
@@ -686,7 +686,11 @@ void WINAPI DrawStatusTextW (HDC hdc, LPCRECT lprc, LPCWSTR text, UINT style)
         border = 0;
 
     oldbkcolor = SetBkColor (hdc, comctl32_color.clrBtnFace);
+#ifdef __REACTOS__
+    DrawEdge (hdc, &r, border, BF_RECT|BF_ADJUST);
```

Comment:
What about sending it to Wine?

Reply:
I do not want to have to deal with Wine, but I am OK with someone else submitting this. Thanks.